docs/man/rpm-version.7.scd

```diff
@@ -0,0 +1,134 @@
+RPM-VERSION(7)
+
+# NAME
+*rpm-version* - RPM version system
+
+# SYNOPSIS
+\[_EPOCH_*:*]_VERSION_[*-*_RELEASE_]
+
+# DESCRIPTION
+A label known as _EVR_ is used to refer to software versions in RPM
```

Comment:
How about:
"A label known as _EVR_ is used to refer to software versions in RPM, and consists of up to three _components_:"

The "one to three" is correct of course but "up to three" reads a bit more naturally, I think. The fact that at least one component is needed is kinda implicit (since an empty EVR makes no sense) :smile: